cgnsutilities/cgnsutilities.py

```diff
@@ -2374,13 +2380,16 @@ def randomize(self, keepRHS):
                     for idim in range(3):
                         self.coords[:, j, k, idim] = self.coords[::-1, j, k, idim]
 
-    def symmZero(self, idir):
+    def symmZero(self, idir, family=None):
         for bc in self.bocos:
-            if bc.internalType == "bcsymmetryplane":
-                # 'r' is the range. We need to subtract off -1 from
-                # the low end since it was in fortran 1-based ordering
-                r = bc.ptRange.copy()
-                self.coords[r[0, 0] - 1 : r[0, 1], r[1, 0] - 1 : r[1, 1], r[2, 0] - 1 : r[2, 1], idir] = 0.0
+            if "symmetry" in bc.internalType:
```

Comment:
There are only two standard CGNS bc types defined that have `symmetry` in the name. I would prefer making this explicit and do a full check and just adding the other options instead of this partial check.

Reply:
Made the check explicit.